src/test/java/RenamePlaylistTest.java

```diff
@@ -0,0 +1,26 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RenamePlaylistTest extends BaseTest {
+
+
+    @Test
+
+    public void renamePlaylist() {
+
+
+        // GIVEN user is on the login page ("https://bbb.testpro.io/");
+        driver.get(url);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226536092-7de5cb29-daf1-4541-b181-ff308d17fb05.png)